src/lib/iptables.ts

```diff
@@ -167,18 +162,17 @@ export function convertToRestoreRulesFormat(rules: Rule[]): string {
 const iptablesRestoreAdaptor: RuleAdaptor = async (
 	rules: Rule[],
 ): Promise<void> => {
-	const rulesFiles = _(rules)
-		.groupBy((rule) => `v${rule.family}`)
-		.mapValues((ruleset) => convertToRestoreRulesFormat(ruleset))
-		.value();
+	const rulesFiles = _.mapValues(
+		Object.groupBy(rules, (rule) => `v${rule.family}`),
+		(ruleset) => convertToRestoreRulesFormat(ruleset!),
+	);
 
 	// run the iptables-restore command...
-	for (const family of Object.getOwnPropertyNames(rulesFiles)) {
+	for (const [family, ruleset] of Object.entries(rulesFiles)) {
```

Comment:
This was the easiest way to satisfy typescript now that it sees `rulesFiles` as an actual type rather than `any`